pkg/kubernetes_clusters.go

```diff
@@ -0,0 +1,16 @@
+package clouddriver
```

Comment:
Move this to package `kubernetes`, so we can reference it as `kubernetes.Cluster`.

Reply:
Sure